app/views/widgets/primary_content/_wysiwyg_anchor_links.html

```diff
@@ -1,50 +1,51 @@
-<div class="wysiwyg-widget editableContent clearfix">
-  <div class="content">
-    <div class="a-z">
-      <h3 class="anchorLinks">
-        <a href="#A" name="top">A</a>&#160;&#160;<a href="#B">B</a>&#160;&#160;<a href="#C">C</a>
-      </h3>
```

Comment:
Removed A here to avoid conflicts with A section in A-to-Z widget I added to this template.